src/ast/data_type.rs

```diff
@@ -98,6 +100,9 @@ impl fmt::Display for DataType {
             DataType::Varchar(size) => {
                 format_type_with_optional_length(f, "CHARACTER VARYING", size, false)
             }
+            DataType::Nvarchar(size) => {
+                format_type_with_optional_length(f, "Numeric CHARACTER VARYING", size, false)
```

Comment:
```suggestion
                format_type_with_optional_length(f, "NUMERIC CHARACTER VARYING", size, false)
```

I think this should be all capitals to be consistent with the rest of this function